database/warehouse/replay/player.py

```diff
@@ -12,10 +12,12 @@
 
 class player(Injectable, Base):
     __tablename__ = "player"
-    __table_args__ = {"schema": "replay"}
+    __table_args__ = ( UniqueConstraint("pid", "info_id", name="pid_info_id_unique")
```

Comment:
We now enforce unique constraint pair pid, info_id pair on player relation. Perhaps it can act as primary_key.